Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css

```diff
@@ -2,105 +2,54 @@
 
 button.webButton
 {
-  background-color: Transparent;
-  cursor: hand;
+  background-color: transparent;
+  border: 1px solid var(--color-primary);
+  border-radius: 2px;
+  padding: 0.6em;
+  box-sizing: border-box;
   cursor: pointer;
-  text-decoration: none !important;
-  border: none;
-  position: relative;
-  display: inline-block;
-  vertical-align: middle;
-  padding: 0px 2px;
-  -moz-padding-start: 0;
-  -moz-padding-end: 0;
-  outline: none;
-}
-
-button.webButton::-moz-focus-inner
-{ 
-  border: 0;
 }
 
-button.webButton span.buttonBody
+button.webButton:hover,
+button.webButton:focus
 {
-  border: solid 1px #AAAAAA;
-  font-weight: bold;
-  background-image: url(../Image/ButtonBackground.gif);
-  background-color: #E6E5E7;
-  background-position: center top;
-  background-repeat: repeat-x;
-  
-  display: block;
-  position: relative;
-  top: 0px;
-  left: -2px;
-  
-  width: 100%;
-  white-space: nowrap;
-
-  padding-top: 2px;
-  padding-bottom: 2px;
-  padding-left: 1px;
-  padding-right: 1px;
+  background-color: var(--color-highlight);
 }
 
-button.webButton span.buttonBody:first-child::before
+button.webButton.primary
 {
-  content: "";
-  height: 1.25em;
-  width: 0;
-  display: inline-block;
-  vertical-align: text-bottom;
-}
-
-button.webButton span.buttonBody img
-{
-  vertical-align: bottom;
+  background-color: var(--color-primary);
+  color: white;
   border: none;
 }
 
-button.webButton:focus span.buttonBody,
-button.webButton:hover span.buttonBody
+button.webButton.primary:hover,
+button.webButton.primary:focus
 {
-  background-image: url(../Image/ButtonBackgroundHover.gif);
-  background-color: #DEECFC;
-  border: 1px solid #7CB2E6;
+  background-color: var(--color-primary-highlight);
 }
 
-button.webButton:focus span.buttonBody
+button.webButton.secondary
 {
-  text-decoration: underline;
+  border: none;
 }
 
-button.webButton:active span.buttonBody
+button.webButton[disabled]
 {
-  background-image: url(../Image/ButtonBackgroundActive.gif);
-  background-color: #C5E0FC;
-  border: 1px solid #569CE0;
+  background-color: transparent;
+  color: var(--color-standard-border);
+  border: 1px solid var(--color-standard-border);
```

Comment:
We need to introduce `--color-disabled-border` for this.

Reply:
Which would be equivalent to `--remotion-themed-border`?

Reply:
I think so? "disabled" is special as it's meant to make a quite uniform gray-style. It will be important to be able to distinguish between just a gray border and a disabled element, so there is also the grayed-out background when there's color and and the gray text-color. Only thing that' not becoming gray (typically) is white background.

Reply:
Added properties in [`3f60642` (#171)](https://github.com/re-motion/Framework/pull/171/commits/3f6064273d44ba50e94ed957dfa8a51abcbe75b0), please check.

Reply:
looks good.